course_content/iris_course/5.Cube_Plotting.ipynb

```diff
@@ -223,7 +223,6 @@
     "    <b><font color=\"brown\">Exercise: </font></b>\n",
     "    <p>Plot the Iris equivalent of the colour blockplot method \n",
     "      <a href=\"https://matplotlib.org/api/_as_gen/matplotlib.pyplot.pcolormesh.html\">matplotlib.pyplot.pcolormesh</a>    for the first timestep of the 'airtemps' data, i.e. <b><font face=\"courier\" color=\"black\">airtemps[0]</font></b>.</p> \n",
-    "    <p>Zoom in on a small region so you can see the individual data points.</p>\n",
```

Comment:
I've removed this bit as we haven't introduced the fact that iris create cartopy geoaxes yet. That is done in the section after this exercise.

Reply:
I'm a bit conflicted about that :  I wanted this to demonstrate what we just said about how a blockplot is different from contouring and why you would use it.  And for that, you really need to see the individual cells ...

Could we "zoom in" by plotting only a small section of the data, instead ??

Reply:
Addressed in #168